helper/Data.php

```diff
@@ -307,6 +307,7 @@ public function buildControllerResponseJson($action, $details = array())
      * @param $amount
      * @param $currency
      * @return string
+     * @deprecated Use Adyen\Util\Currency sanitize() from Adyen php api library
```

Comment:
Add readme that in the next major release the deprecated functions will be removed

Reply:
Add "will be removed in v2.0.0"